source/common/common/key_value_store_base.h

```diff
@@ -24,17 +27,20 @@ class KeyValueStoreBase : public KeyValueStore,
   // parses key value pairs from |contents| and inserts into store_.
   // Returns true on success and false on failure.
   bool parseContents(absl::string_view contents);
+  void ttlCallback(const std::vector<std::string>& keys);
 
   std::string error;
```

Comment:
Admittedly, this seems to be pre-existing, but as long as you're here, can you remove this?